backend/benefit/common/utils.py

```diff
@@ -143,7 +321,9 @@ def days360(start_date, end_date):
     )
 
 
-def duration_in_months(start_date, end_date, decimal_places=None):
+def duration_in_months(
+    start_date: date, end_date: date, decimal_places: Union[int, None] = None
```

Comment:
There's also `Optional` from `typing` for these cases. Doesn't really matter, as I hope we can upgrade to python 3.10 soon.